runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -197,8 +213,9 @@ typedef struct nnfw_train_info
   float learning_rate = 0.001f;
   /** Batch size */
   uint32_t batch_size = 1;
-  /** loss type */
-  NNFW_TRAIN_LOSS loss = NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR;
+  /** loss info */
+  nnfw_loss_info loss_info{.loss = NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR,
```

Comment:
It should be an `INVALID` loss type. But that is not the scope of this PR. I will make another pr for this.